lib/connection.js

```diff
@@ -1388,6 +1388,21 @@ Connection.prototype.syncIndexes = async function syncIndexes() {
  * Switches to a different database using the same connection pool.
  *
  * Returns a new connection object, with the new db.
+ * 
+ * ####Example
+ *    async function createMultiTenatConnection(tenatId, modelName, schema) {
+  *     const conn = await mongoose.createConnection('mongodb://localhost:27017/test');
```

Comment:
Why is this line here? `conn` isn't used at all here.

Reply:
I thought making a connection is necessary for better understanding of reader.